monkey/monkey_island/cc/next_ui/src/middleware.ts

```diff
@@ -9,15 +9,25 @@ const PROTECTED_PATHS_FOR_AUTHORIZED_USER: string[] = [
     AUTH_PATHS.SIGN_UP
 ];
 
+// if pathname starts with any of the paths in PROTECTED_PATHS_FOR_AUTHORIZED_USER
+// return true
```

Comment:
Instead of writing code and explaining what it does in the comments, we should strive to write readable code that doesn't require explanation.

Reply:
You are right. The code is already self-explanatory.